### ISSUE TITLE
Unresolved constraint when trying to add package tap:i18n

### ISSUE THREAD

**Issue by lowi** [meteor/meteor#9869]
With the latest development version of Meteor (Tested with version of meteor: 9fc0251eeeaea096113743e9ebd68fe50a714118 - Mon May 7 17:02:25 2018), I can't add tap:i18n package due to an unresolved constraint with coffeescript:

```
➜  test git:(master) ✗ ../meteor/meteor add tap:i18n 
 => Errors while adding packages:             
                                              
While selecting package versions:
error: Conflict: Constraint coffeescript@1.0.4 is not satisfied by coffeescript 2.2.1_1.
Constraints on package "coffeescript":
* coffeescript@=2.2.1_1 <- top level
* coffeescript@1.0.4 <- tap:i18n 1.8.2
```

To reproduce the problem, I ran the following command:
```
meteor create --bare test
cd test
meteor add tap:i18n
```

I have created a repo on which `meteor add tap:i18n` will fail:
https://github.com/lowi/meteor_1.6.2_break_coffeescript

I noticed this issue beginning of March when I was investigating some minifiying issue.

**Comment by lowi** [comment]
tap:i18n is referenced in guides.meteor.com, and I am using it with tap:i18n-db.
If I can avoid upgrading at the moment, it would helpful.

As for the error linked to coffeescript, I am not sure if it's a tap:i18n issue, or rather a meteor build issue.

**Comment by jorgeer** [comment]
I started having other issues with tap:i18n recently, it spent half a minute parsing my language files for example. I switched to the npm i18next which tap is built on instead, and imported the language files manually. Works like a charm. I believe i18next has most of the features of the tap package in any case?

Regarding the coffeescript package, I think it's just a matter of the package being outdated and not accepting newer versions of coffeescript. Is using the coffeescript version tap:i18n wants an option?

**Comment by lowi** [comment]
Seems like a good alternative to use directly i18next.

I didn't find any constraint on coffeescript version in tap:i18n package.

**Comment by aboire** [comment]
it's related to the test practicalmeteor:mocha which is on older version of coffeescript

https://blog.meteor.com/announcing-meteor-1-6-1-50aad71da4e6

"Speaking of CoffeeScript, while Meteor 1.6.1 was under development, a new major version of the coffeescript and coffeescript-compiler packages were released (most recently 2.0.3_4), which is worth mentioning for the same reason as the babel-compiler@7.0.0 upgrade: packages like practicalmeteor:mocha that depended on a 1.x version of coffeescript may need to be updated or replaced. Specifically, we have found meteortesting:mocha to be a good choice."
